biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
```

Comment:
Sounds great, but why is it necessary to follow the standard? I think being clear on the reason helps to understand why this is helpful.

Reply:
true. suggestion ;) ?

Reply:
Unfortunately not. Because I didn't know what's meant by standard here. That's why I asked ;)

Reply:
Unfortunately not. Because I don't know what's meant by standard here. That's why I asked ;)